scripts/test.js

```diff
@@ -26,16 +26,17 @@ var passingTests = [
   'node lib/cli.js spec/plugins/browserGetUnsyncedConf.js',
   'node lib/cli.js spec/plugins/waitForAngularConf.js',
   'node lib/cli.js spec/interactionConf.js',
-  'node lib/cli.js spec/directConnectConf.js',
+  //'node lib/cli.js spec/directConnectConf.js',
```

Comment:
These are working on my machine - lets leave them in, and we can figure out why your machine doesn't like them (or maybe we're missing some setup instructions that I've forgotten about).